apps/test-gimbal/main.cpp

```diff
@@ -0,0 +1,34 @@
+#include "mbed.h"
+
+#include "Encoder.h"
+#include "EncoderAbsolute_PWM.h"
+#include "ActuatorController.h"
+#include "CANMsg.h"
+#include "CANBuffer.h"
+#include "Servo.h"
+#include "PwmOut.h"
+
+#define TIME_ERROR_FACTOR 1
+
+// Init. Components
+
+PwmOut tilt (PB_15);
+PwmOut pan (PB_14);
+
+int max_width_us = 2.1*1000*TIME_ERROR_FACTOR,

```

Comment:
Above TIME_ERROR_FACTOR is defined as 1. I may be misunderstanding something but from this code I don't see why multiplying by 1 is useful.